repose-aggregator/src/docs/asciidoc/services/datastores.adoc

```diff
@@ -231,16 +221,15 @@ The following is a basic sample configuration.
 ----
 
 ==== Full Configuration
-
 [source,xml]
 .remote-datastore.cfg.xml
 ----
 <?xml version="1.0" encoding="UTF-8"?>
 <remote-datastore>
-    <cluster id="repose"                             <!--1-->
-             host="remote.example.com"               <!--2-->
-             port="8080"                             <!--3-->
-             useHTTPS="true"                         <!--4-->
+    <cluster id="repose" <!--1-->
+             host="remote.example.com" <!--2-->
+             port="8080" <!--3-->
+             useHTTPS="true" <!--4-->
```

Comment:
Actually, this was intentional. After I showed @adrianjgeorge another place where they were lined up, it was hated so much I went and changed them all for consistency.